cosmwasm/ibc-union/core/src/contract.rs

```diff
@@ -1897,7 +1894,11 @@ fn send_packet(
     timeout_timestamp: Timestamp,
     data: Vec<u8>,
 ) -> ContractResult {
-    if timeout_timestamp.is_zero() && timeout_height == 0 {
+    if timeout_height != 0 {
+        return Err(ContractError::TimeoutHeightUnsupported);
+    }
```

Comment:
would be good to have a test for this such that we don't accidentally accept non-zero heights

Reply:
this is just informational the semantic has been removed from the receiving/timeout side so it's effectively a noop even if it passes